tests/util/xcm.ts

```diff
@@ -435,3 +792,24 @@ export class XcmFragment {
     return this;
   }
 }
+
+function replaceNetworkAny(obj: AnyObject | Array<AnyObject>) {
+  if (Array.isArray(obj)) {
+    return obj.map((item) => replaceNetworkAny(item));
+  } else if (typeof obj === "object" && obj !== null) {
+    const newObj: AnyObject = {};
+    for (const key in obj) {
+      if (key === "network" && obj[key] === "Any") {
+        newObj[key] = "Ethereum";
```

Comment:
We should not replace "Any" by "Ethereum" but by null instead

Reply:
That's not a valid enum
```
'ByGenesis' | 'ByFork' | 'Polkadot' | 'Kusama' | 'Westend' | 'Rococo' | 'Wococo' | 'Ethereum' | 'BitcoinCore' | 'BitcoinCash';
```

Reply:
Oh you mean null the whole network field entirely

Reply:
Yes, it doesn't work as expected if we use "Ethereum" by default. Using null works fine.